docs/rules.md

```diff
@@ -10,7 +10,8 @@ We should try and list out all the validation rules we know of by file type. E.g
 | Done? | MsgType | Rule name | Addon type | Description | File Type | Source ref | Old Code | New Code |
 | ----- | ------- | --------- | ---------- | ----------- | --------- | ---------- | -------- | -------- |
 | :x: | warning | dangerous_contract | | Dangerous XPCOM contract ID |  | [testcases/javascript/actions.py](https://github.com/mozilla/amo-validator/blob/master/validator/testcases/javascript/actions.py)|
-| :x: | warning |  banned_identifier | | Banned or deprecated JavaScript Identifier | | | | |
+| :white_check_mark: | warning |  banned_identifier | | Banned or deprecated JavaScript Identifier | | | | BANNED_NEWTHREAD |
```

Comment:
I wonder actually if our current approach makes sense for this. It would be nice to just be able to add to the list of BANNED_IDENTIFERS rather than add a whole load of other things. But then that's where const bites us because you can't dynamically define constants. Maybe we should change that. Let's chat about it tomorrow.